test/integration/character_can_get_rid_of_incidents_test.rb

```diff
@@ -35,4 +35,29 @@ class CharacterCanGetRidOfIncidentsTest < ActionDispatch::IntegrationTest
       assert page.has_content?("Health: 10")
     end
   end
+
+  test "character can see incidents in apothocary or inn" do
+    create_start_of_game
+    ApplicationController.any_instance.stubs(:in_game).returns(true)
+    item = create(:item)
+    c1, c2 = create_list(:category, 2)
```

Comment:
Useless assignment to variable - `c2`. Use `_` or `_c2` as a variable name to indicate that it won't be used.